sound/soc/sof/ipc.c

```diff
@@ -405,7 +438,9 @@ void snd_sof_ipc_msgs_rx(struct snd_sof_dev *sdev)
 	case SOF_IPC_GLB_COMPOUND:
 	case SOF_IPC_GLB_TPLG_MSG:
 	case SOF_IPC_GLB_PM_MSG:
+		break;
 	case SOF_IPC_GLB_COMP_MSG:
+		ipc_comp_notification(sdev, &hdr);
```

Comment:
@singalsu @ujfalusi Just to double-check, I assume old firmware versions will just not send these? Scenario in my mind was that if some old firmware sent these IPCs and whether kernel would handle messages (from the old FW) correctly. But based on my understanding, this seems fine.

Reply:
@kv2019i, yes, that is correct: old and current FW does not send these messages. The kernel prior to this patch would ignore the messages.